.env.example

```
@@ -1,24 +1,17 @@
-# Required for MyCarrierPortal API authentication
+ Required for MyCarrierPortal API authentication
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The Required for MyCarrierPortal API authentication key has incorrect delimiter

(IncorrectDelimiter)